src/utils/trust.details.ts

```diff
@@ -147,23 +162,37 @@ export const postTrustDetails = async (req: Request, res: Response, next: NextFu
     }
 
     //  if present, get existing trust from session (as it might have attached trustees)
-    const trust = getTrustByIdFromApp(appData, details.trust_id);
+    let trust;
+    if (isReview) {
+      trust = getReviewTrustById(appData, details.trust_id);
+    } else {
+      trust = getTrustByIdFromApp(appData, details.trust_id);
+    }
     Object.keys(details).forEach(key => trust[key] = details[key]);
 
     //  update trust  in application data at session
-    appData = saveTrustInApp(appData, trust);
+    if (isReview) {
+      appData = updateTrustInReviewList(appData, trust);
+    } else {
+      appData = saveTrustInApp(appData, trust);
```

Comment:
saveTrustInAppData

Reply:
same here @AndreKramer7654 its an already existing method we've just wrapped in the if/else